sdk/eventhub/azure-eventhub/CLIENT_DEVELOPER.md

```diff
@@ -0,0 +1,201 @@
+# Azure Event Hubs Client Library Developer Guide
+
+This guide is intended for developers contributing to the Azure Event Hubs Python client library. It provides information on setting up your development environment, running tests, and contributing to the codebase.
+
+## Setting Up Development Environment
+
+### Prerequisites
+
+- Python version supported by the client library
```

Comment:
link out to this document to see the min supported python version https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy